hc-nested-list-behavior.html

```diff
@@ -132,16 +132,22 @@
         } else items[i].open = false; // If it's not the item we want, ensure it's closed
       }
 
-      // fetch children and update when done.
-      const ids = Object.keys(parts);
-      this.fetchChildren(item, parts, (parts) => {
-        const newIds = Object.keys(parts).filter((id) => ids.indexOf(id) < 0);
-        for (let i = 0; i < newIds.length; i++) {
-          const id = newIds[i];
-          for (let j = 0; j < parts[id].length; j++) {
-            parts[id][j] = this.__setDefaultPart(parts[id][j]);
-          }
-        }
+      (new Promise((res) => {
+        if (item) {
```

Comment:
You would have to:
```
if (!item) {
  res(parts);
  return;
}
```
in order to keep the block of code after it from running, since that's the case I opt'd for if/else instead of the extra code to inject a return.